From d26faf7970a0c7ed9342297be29773bcd1722d72 Mon Sep 17 00:00:00 2001
From: David Neto <dneto@google.com>
Date: Tue, 25 Jul 2023 11:58:49 -0400
Subject: [PATCH] cmake: Remove redundant -std=c++11 flag

This is already covered by the CMAKE_CXX_STANDARD directive
---
 cmake/utils.cmake | 1 -
 1 file changed, 1 deletion(-)

diff --git a/cmake/utils.cmake b/cmake/utils.cmake
index d64757dd6..e9475b9cf 100644
--- a/cmake/utils.cmake
+++ b/cmake/utils.cmake
@@ -59,7 +59,6 @@ endfunction(shaderc_default_c_compile_options)
 function(shaderc_default_compile_options TARGET)
   shaderc_default_c_compile_options(${TARGET})
   if (NOT "${MSVC}")
-    target_compile_options(${TARGET} PRIVATE -std=c++11)
     if (NOT SHADERC_ENABLE_SHARED_CRT)
       if (WIN32)
         # For MinGW cross compile, statically link to the C++ runtime.
